docs/getting-started/generating-a-mnemonic.en.md

```diff
@@ -39,6 +51,8 @@ Krux keeps track of every roll you enter and displays the cumulative string of o
 
 When you have entered your final roll, Krux will hash this string using SHA256 and output the resulting hash to the screen so that you can verify it for yourself.
 
+In the case a camera snapshot is used as source, image bytes, which contain pixels data in RGB565 format, will be hashed just like it is done with the dice rolls string.
```

Comment:
Something like [this](https://photos.app.goo.gl/29j64WVVbeBR7D578) could be done. But I can't affirm pixel colors and lightness variance are correlated to entropy's quality.
Here I was measuring 3 standard deviations (square root of variances):
l_stdev = lightness standard deviation
a_stdev = green–magenta standard deviation
b_stdev = blue–yellow standard deviation

Reply:
So as not to hold up this PR any longer, could you mark the feature experimental until someone with experience in this domain can confirm that a single image will always provide sufficient entropy? My hunch is it won't, at least not all of the time, and I think Krux should have a way to indicate that to the user.

Entropy doesn't necessarily equal information. I can "generate" a 256 bit number from my mind, but it won't have 256 bits of entropy since my internal RNG isn't very good. I think pictures are similar, just because they have many bits of information doesn't mean that those bits are entropy.

Reply:
Sorry just saw this now. I'll add a "experimental" tag on intro message.
Sorry I broke the link of experiments I made to estimate poor entropy. You can [try this post](https://twitter.com/odudex/status/1699126162650275945?s=20), from which we got good feedback. But this "detection" other improvements are things to be discussed for later releases.